bootstrap/helpers/parsers.php

```
@@ -289,24 +289,20 @@ function parseDockerVolumeString(string $volumeString): array
             $varName = $parts[0];
-            $defaultValue = isset($parts[1]) ? $parts[1] : '';
-
-            // If there's a non-empty default value, use it for source
+            $defaultValue = $parts[1] ?? '';
             if ($defaultValue !== '') {
                 $source = $defaultValue;
             } else {
-                // Empty default value, keep the variable reference for env resolution
+                // Empty default — clean up to simple variable reference
                 $source = '${'.$varName.'}';
             }
```

PHPMD (2.15.0)

[error] The method parseDockerVolumeString uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)